src/status_im/contexts/wallet/collectible/tabs/overview/view.cljs

```diff
@@ -0,0 +1,73 @@
+(ns status-im.contexts.wallet.collectible.tabs.overview.view
+  (:require
+    [clojure.string :as string]
+    [quo.core :as quo]
+    [quo.foundations.resources :as quo.resources]
+    [quo.theme]
+    [react-native.core :as rn]
+    [status-im.contexts.wallet.collectible.tabs.overview.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn- traits-section
+  [traits]
+  (when (pos? (count traits))
+    [rn/view
+     [quo/section-label
+      {:section         (i18n/label :t/traits)
+       :container-style style/traits-title-container}]
+     [rn/flat-list
+      {:render-fn               (fn [{:keys [trait-type value]}]
+                                  [quo/data-item
+                                   {:description     :default
+                                    :card?           true
+                                    :status          :default
+                                    :size            :default
+                                    :title           trait-type
+                                    :subtitle        value
+                                    :container-style style/traits-item}])
```

Comment:
I've always had a quesiton about this `render-fn` parameter.

Are we turning this function into a reagent component? if so, that is like creating a component on the fly in each re render, that may lead to bugs or slow performance.

The orginal version of flat list (from react native) expects a function returning a react element, since we are not using `reagent/as-element` here, I guess that is something being done behind our implementation, but I'm wondering the way that is being handled.

Maybe @flexsurfer or @ilmotta know more abour it

Reply:
This code was there already, I just moved it to a structured folder. I will refactor it as requested 👍

Reply:
@ulisesmac, I had the same question as you a long time ago, but never really tried refactoring. I see you made a comment in another PR about this, I answered a little bit there https://github.com/status-im/status-mobile/pull/18281#discussion_r1435245875